### DISCUSSION TITLE
Which dates to consider for the CFP

### DISCUSSION THREAD

**Discussion by mcjaeger** [software-composition-analysis/fosdem-2022-devroom#7] [General]
What would be the current proposal of timelines i terms of which dates shall be in the dev room call?

**Comment by amottier** [comment]
Just as a reminder for my self: we agree on in our call yesterday that December 19 is the CfP submission deadline.
I include in the readme a deadline for us to get back to submitter on December 27.
We don't not yet about the deadline for video recording submission but I include it based on what happened last year to give an heads up to submitter.
